.pre-commit-config.yaml

```diff
@@ -22,9 +22,10 @@ repos:
   hooks:
   - id: isort
 - repo: https://github.com/asottile/setup-cfg-fmt
-  rev: v1.9.0
+  rev: v1.14.0
   hooks:
   - id: setup-cfg-fmt
+    args: ["--min-py3-version", "3.5"]
```

Comment:
Can you expand on this? What does it mean and why is it needed?

I want to keep the project buildable with python2, for distro workflows. Otherwise I'd have to have a dependency on Python 3 to build `build` for Python 2.

Reply:
This is just metadata for setup-cfg-fmt, of what's the minimum python 3 version, default is python 3.6 for it. So we're overwriting that as we still support 3.5.

Reply:
Ah, okay :+1: